_docs/installation.md

```diff
@@ -21,15 +21,15 @@ Before you install, please read our [Contributing]({{ site.baseurl }}/docs/contr
 
 ## System Requirements
 
-> Note: As of 0.1.0-beta.3, Flarum utilizes [Composer](https://getcomposer.org) to manage its dependencies and extensions. This means that **Flarum cannot be installed on hosts without SSH (command-line) access**. Rest assured that we are planning to address this in the future and make sure that Flarum is accessible to everyone. In the meantime, if you want to run Flarum, you will need to find a host that permits SSH access.
```

Comment:
I like the explicitness of this:

> Flarum utilizes [Composer](https://getcomposer.org) to manage its dependencies and extensions. This means that **Flarum cannot be installed on hosts without SSH (command-line) access**.

Can we please retain this?

Reply:
Ack!